dtm0/it/all2all/m0crate.yaml

```diff
@@ -0,0 +1,35 @@
+CrateConfig_Sections: [MOTR_CONFIG, WORKLOAD_SPEC]
+MOTR_CONFIG:
+   MOTR_LOCAL_ADDR:  10.230.248.89@tcp:12345:4:1
```

Comment:
Just curious question : is there a reason behind MOTR_LOCAL_ADDR and MOTR_HA_ADDR to be hardcoded?

Reply:
Yes, they need to be specified. Another solution is to use a template that is pre-processed before running of m0crate (like it is done by io_workload script), but I would say that it may be treated as further improvements.

Reply:
Ok. May be we should make note of it somewhere.. or have a readme so the test runner knows that they need to specify these IP addresses in m0crate.yaml file ? If it (instruction/Note) is  already available, please ignore the comment.

Reply:
Good point, added README.md.